### ISSUE TITLE
How to detect overfitting in WandB ?

### ISSUE THREAD

**Issue by Ca-ressemble-a-du-fake** [DigitalPhonetics/IMS-Toucan#91]
Hi,

I am looking for a way to detect overfitting in WandB so that I know when to stop the training when I am away from the GPU.

Are there some clues that can be seen on the spectrogram that show it is time to stop the training ?

Sorry for this whole bunch of questions, feel free to answer them at your pace :wink: ! 

Edit : I've just noticed all the loss graphs that I hadn't seen earlier. The finetuning stopped at 6000 steps but the losses want to keep decreasing. So that may be a good hint, as long as the losses are decreasing there may be a benefit of pursuing the training. Does this make sense ? 

Thanks in advance for your answer,

**Comment by Flux9665** [comment]
yes, you can observe the losses and see when the curve flattens out. Any weird non-smooth spikes or drops in the loss curve can indicate that the model starts overfitting.

**Comment by Ca-ressemble-a-du-fake** [comment]
Ok that makes sense, thank you !
